zerver/openapi/zulip.yaml

```diff
@@ -11981,6 +11984,17 @@ paths:
                   items:
                     type: integer
                   example: [1, 2]
+                subscribe_to_default_streams:
+                  description: |
+                    Boolean indicating whether to subscribe the newly created user
+                    to default streams in the organization.
```

Comment:
Maybe clarify that this is "Boolean indicating that the newly created user should be subscribed to exactly the default streams for the organization.", maybe with "default streams" as a /help/ link.

Reply:
I guess we need to remove the "exactly" language given the new behavior.